html/elements/keygen.json

```diff
@@ -14,7 +14,7 @@
               "version_removed": "57"
             },
             "edge": {
-              "version_added": true
+              "version_added": "≤18"
```

Comment:
I think we need to add` "version_removed": "79"` here.

Reply:
Oops!  I must have missed this scenario where Chrome has a `version_removed` value and there was support within EdgeHTML.  I'll try to programmatically fix this case so all the mirroring PRs will be fixed!